R/verb-arrange.R

```diff
@@ -1,3 +1,28 @@
+#' Arrange rows by variables in a local lazy tibble
+#'
+#' Order tbl_lazy rows by an expression involving its variables.
+#'
+#' @section Missing values:
+#' When applied to `tbl_lazy`, the [arrange()] method sorts NAs first (at the
+#' beginning) compared to the default sort behavior on data frames, which places
+#' NA values last (at the end).
+#'
+#' @inheritParams dplyr::arrange
+#' @return An object of the same class as `.data`.
+#' @examples
+#' library(dplyr)
+#'
+#' dbplyr::memdb_frame(a = c(3, 4, 1, 2)) %>%
+#'   arrange(a)
+#'
+#' # tbl_lazy: NAs sorted first
+#' dbplyr::memdb_frame(a = c(3, 4, NA, 2)) %>%
+#'   arrange(a)
+#'
+#' # tbl_df: NAs sorted last
```

Comment:
I wouldn't show this, but instead show `arrange(is.na(a), a)` to force NAs to sort first.

Reply:
Fixed [here](https://github.com/tidyverse/dbplyr/pull/222/commits/6873d6482a1cfa076b4d49d0787485ab996cc3cd#diff-12fbbf8bf4196bb24a276dff57ec3116R22)